styles/styles.css

```
@@ -98,3 +98,251 @@ body {
 .advantages-item:nth-child(4) {
   background-color: rgba(131, 179, 211, 0.2);
 }
+
+.service {
+  text-align: center;
+  color: rgb(0, 0, 0);
+}
+
+.service-lead {
+  font-size: 30px;
+  font-weight: 700;
+  line-height: 36px;
+  text-transform: uppercase;
+}
+
+.service-description {
+  color: rgb(51, 51, 51);
+  font-size: 22px;
+  line-height: 26px;
+}
+
+.service-item {
+  background-color: rgba(131, 179, 211, 0.12);
+}
+
+.service-item:nth-child(2) {
+  background-color: #ffffff;
+}
+
+.service-item h3 {
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 28px;
+  text-transform: uppercase;
+}
+
+.service-item p {
+  color: rgb(51, 51, 51);
+}
+
+.proposal {
+  text-align: center;
+  color: rgb(51, 51, 51);
+}
+
+.proposal-lead {
+  color: rgb(0, 0, 0);
+  font-size: 30px;
+  font-weight: 700;
+  line-height: 36px;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.proposal-description {
+  font-size: 22px;
+  line-height: 26px;
+}
+
+.proposal-button {
+  background: rgb(117, 97, 87);
+  color: rgb(255, 255, 255);
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 36px;
+  text-transform: uppercase;
+  border: none;
+}
+
+.subscribe {
+  background-color: #000;
+  background-image: url(../images/subscribe.png);
```

если не ошибаюсь, что заработало нужно взять изображение в кавычки
```suggestion
  background-image: url("../images/subscribe.png");
```

---

без кавычек тоже работает